### PR TITLE
Feature/serv 627 remove nfs

### PR DESCRIPTION
**Pull request by turegjorup**
- Removed `nfs` usage in templates and script
- Bumped version number for breaking change
- Added Changelog

### CODE REVIEW

File: templates/drupal-7/docker-compose.yml [hunk 1]
```diff
@@ -54,8 +55,8 @@ services:
     ports:
       - '8080'
     volumes:
-      - ${PWD}/.docker/vhost.conf:/etc/nginx/conf.d/default.conf:ro
-      - nfsApp:/app:delegated
+      - ./.docker/vhost.conf:/etc/nginx/conf.d/default.conf:ro
```
Comment (rimi-itk) [inline templates/drupal-7/docker-compose.yml:58]: 👍

File: templates/aakbcms/docker-compose.yml [hunk 2]
```diff
@@ -42,7 +42,7 @@ services:
     depends_on:
       - mariadb
     volumes:
-      - nfsApp:/app:delegated
+      - .:/app
```
Comment (rimi-itk) [inline templates/aakbcms/docker-compose.yml:45]: 👍

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+## [Unreleased]
+
+### Removed
+
+- Removed `nfs` volume usage for better MacOS Ventura compatibility
+
+### Fixed
+
+- Fixed foldouts in Dutch translation
```
Comment (rimi-itk) [inline CHANGELOG.md:16]: Really?

**Review by rimi-itk — APPROVED** [review]
Approved with a comment on the Dutch translation.

File: CHANGELOG.md (hunk 3 above)
Comment (turegjorup) [reply CHANGELOG.md:16]: Whops 🙄

**Comment by cableman** [comment]
It would be nice to have "NFSDisable/NFSClean" up to remove the share on the macs....

**Pull request merged**